config/locales/vi.yml

```diff
@@ -0,0 +1,3 @@
+vi:
+  hello:
+  title:
```

Comment:
indenting 2 spaces and add end line, please

Reply:
done